views/editor.dt

```diff
@@ -6,16 +6,17 @@ block head
 	link(rel="stylesheet", href="/static/lib/codemirror/addon/lint/lint.min.css")
 	link(rel="stylesheet", href="/static/lib/codemirror/theme/elegant.css")
 	link(rel="stylesheet", href="/static/lib/grid12.min.css")
+	link(rel="stylesheet", href="/static/css/editor.css")
 
 block content
 	.row(ng-controller="DlangTourAppCtrl as ctrl", ng-init="initEditor('#{sourceCode}')")
-		div#tour-content(ng-show="showProgramOutput", ng-class="{'col-md-12': !showSourceCode, 'col-md-6 col-sm-12': showSourceCode}")
```

Comment:
This was wrongly copied over from the tour. The output currently appears on top and thus only half of the width is used for large screens:

![image](https://user-images.githubusercontent.com/4370550/27620860-e913c7f2-5bcc-11e7-9bc9-5e3a731a0839.png)

(we can always decide to change the layout - after all the editor subfeature is 'alpha' anyhow).